evalml/utils/gen_utils.py

```diff
@@ -609,18 +609,19 @@ def are_ts_parameters_valid_for_split(
             smallest_split_size (int): Smallest split size given n_obs and n_splits.
             max_window_size (int): Max window size given gap, max_delay, forecast_horizon.
     """
-    split_size = n_obs // (n_splits + 1)
+    eval_size = forecast_horizon * n_splits
+    train_size = n_obs - eval_size
     window_size = gap + max_delay + forecast_horizon
     msg = ""
-    if split_size <= window_size:
+    if train_size <= window_size:
         msg = (
             f"Since the data has {n_obs} observations and n_splits={n_splits}, "
-            f"the smallest split would have {split_size} observations. "
-            f"Since {gap + max_delay + forecast_horizon} (gap + max_delay + forecast_horizon) >= {split_size}, "
+            f"the smallest split would have {train_size} observations. "
```

Comment:
Isn't `train_size` influenced by `forecast_horizon`? Should we update line 618 to mention `forecast_horizon` as well?